alf/utils/common.py

```diff
@@ -1440,11 +1440,34 @@ def rsync(src, target, includes):
     if alf_dirname != "alf":
         os.system("mv %s/%s %s/alf" % (dest_path, alf_dirname, dest_path))
 
+    # compress the snapshot repo into a ".tar" file
+    os.system("cd %s; tar -czf alf.tar alf" % dest_path)
+    os.system("rm -rf %s/alf" % dest_path)
+
+
+def unzip_alf_snapshot(root_dir: str):
+    """Restore an ALF snapshot from a job directory by unzipping the snapshot
+    'tar' file.
+
+    Args:
+        root_dir: the tensorboard job directory
+    """
+    alf_zipped_repo = os.path.join(root_dir, "alf.tar")
+    if os.path.isfile(alf_zipped_repo):
+        info("=== Using an ALF snapshot at '%s' ===", alf_zipped_repo)
+    else:
+        info("=== Didn't find a snapshot; using update-to-date ALF ===")
+        return
+
+    os.system("rm -rf %s/alf" % root_dir)
+    os.system("cd %s; tar -xzf alf.tar" % root_dir)
```

Comment:
We usually named it `alf.tar.gz` to suggest it is "zipped".

Reply:
> We usually name it `alf.tar.gz` to suggest it is "zipped".

updated the extension